### PR TITLE
added new project by Areg

### PR DESCRIPTION
**Pull request by AregSargsyan**

### CODE REVIEW

**Review by rafayel23 — DISMISSED** [review]
Shat Lavna

File: Areg/tabs/src/app/tab/tab.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+@Component({
+  selector: 'app-tab',
+  templateUrl: './tab.component.html',
+  styleUrls: ['./tab.component.css']
+})
+export class TabComponent implements OnInit {
+ 
+divName = 'News'; // by default
+
+funk(pressedButton: string): void{
```
Comment (vahemadatyan) [inline Areg/tabs/src/app/tab/tab.component.ts]: funkciayi anuny aveli imastalic ban poxi

File: Areg/tabs/src/app/tab/tab.component.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<app-tabs (pressedButton)="funk($event)"></app-tabs>
+
+
+<div class="tabcontent" *ngIf="divName=='Home'" style="background-color:red">
```
Comment (vahemadatyan) [inline Areg/tabs/src/app/tab/tab.component.html]: nenc ara vor unenas mi hat tabContent div 4 hati texy.

File: Areg/tabs/src/app/tab/tab.component.ts [hunk 3]
```diff
@@ -9,13 +9,31 @@ export class TabComponent implements OnInit {
...
+]
+
+tubTextbuilder(ind:number){
+  
+  this.tabcontent= this.tabContent[ind]["name"]
+  this.tabcolor=this.tabContent[ind]["color"];
+  
+}
+
+openTub(pressedButton: string): void{
```
Comment (vahemadatyan) [inline Areg/tabs/src/app/tab/tab.component.ts:29]: openTab*

**Review by vahemadatyan — APPROVED** [review]

**Pull request merged**